Gemfile.lock

```diff
@@ -0,0 +1,251 @@
+PATH
```

Comment:
I think it can be part of the git repo, as long as it's excluded from the gem bundle that gets installed.